### PR TITLE
VAR-SOM-MX8 support 2nd pass

### PR DESCRIPTION
**Pull request by cbrake**

### CODE REVIEW

**Comment by cbrake** [comment]
rebased everything on the latest meta-varscite-bsp and yoe/master and doing a test build now.

**Comment by cbrake** [comment]
getting a freertos-variscite compile error:

```
DEBUG: Executing shell function do_compile                                                                                                                                                                                
find: ‘/scratch/treehaven/yoe-distro/build/tmp/work/imx8qxp_var_som-yoe-linux/freertos-variscite/2.9.x-r0/git/boards/som_mx8qx/demo_apps/disable_cache/hello_world’: No such file or directory                            
WARNING: /scratch/treehaven/yoe-distro/build/tmp/work/imx8qxp_var_som-yoe-linux/freertos-variscite/2.9.x-r0/temp/run.do_compile.1587570:150 exit 1 from 'find /scratch/treehaven/yoe-distro/build/tmp/work/imx8qxp_var_som-yoe-linux/freertos-variscite/2.9
```

**Comment by cbrake** [comment]
removed freertos-variscite from image for now -- can't imagine we'll need that any time soon.

**Comment by cbrake** [comment]
yoe-simple-image boots:

`Yoe Linux 2022.09-rc.1 imx8qxp-var-som ttyLP3`

so at this point, only `yoe-kiosk-image` is failing due to the gstreamer issue.

File: .gitmodules [hunk 1]
```diff
@@ -62,3 +62,6 @@
 	path = sources/poky
 	url = https://github.com/yoedistro/poky
 	branch = master
+[submodule "sources/meta-variscite-bsp"]
+	path = sources/meta-variscite-bsp
+	url = https://github.com/YoeDistro/meta-variscite-bsp.git
```
Comment (kraj) [inline .gitmodules:67]: I think we need to specify `branch = yoe/master` as well.

Comment (cbrake) [reply .gitmodules:67]: thanks, fixed.

**Comment by cbrake** [comment]
after merging kraj/master, yoe-kiosk-image boots and weston runs.

**Comment by kraj** [comment]
@cbrake can you rebase this PR on top of latest `master` and re-push ?

**Comment by cbrake** [comment]
@kraj I think this should be ready to go -- let me know if you see any problems.

Thanks!

**Review by kraj — APPROVED** [review]
lgtm, lets see what CI has to say

**Pull request merged**